### ISSUE TITLE
Floating point issues

### ISSUE THREAD

**Issue by fedemengo-genuino** [xxczaki/cashify#17]
I wrote this simple script using version `2.5.0`
```js
const { convert } = require('cashify')

const rates = {
	"USD": 1,
	"EUR": 0.8235,
}

console.log(convert(1, {from: "USD", to: "EUR", rates}))
```
And the output is `0.8234999999999999`. I was expecting to received `0.8235`, if I run `
console.log(convert(10, {from: "USD", to: "EUR", rates}))` I get a nice value of `8.235`.

**Comment by fedemengo-genuino** [comment]
@xxczaki do you have any idea what could be wrong?

**Comment by Richienb** [comment]
When converting currencies, the input value is automatically multiplied by 100 then divided by 100 after the computation:

https://github.com/xxczaki/cashify/blob/530857c4ceaf98239c9fc45739e511aa8e8c7ea8/src/convert.ts#L26-L29

Because of a JavaScript quirk, `82.35 / 100 === 0.8234999999999999`. To fix this problem, a library like `big.js` can be used until the [decimal proposal](https://github.com/tc39/proposal-decimal) is completed:

```diff
import getRate from './lib/get-rate';
import {Options} from './lib/options';
import parse from './utils/parser';
+import Big from 'big.js';

/**
 * Function, which converts currencies based on provided rates.
 *
 * @param {number | string} amount - Amount of money you want to convert.
 * @param {Object} options - Conversion options.
 * @return {number} Conversion result.
 *
 * @example
 * const rates = {
 * 	GBP: 0.92,
 * 	EUR: 1.00,
 * 	USD: 1.12
 * };
 *
 * convert(10, {from: 'EUR', to: 'GBP', base: 'EUR', rates}); //=> 9.2
 */
export default function convert(amount: number | string, {from, to, base, rates}: Options): number {
	// If provided `amount` is a string, use parsing
	if (typeof amount === 'string') {
		const data = parse(amount);

-		return (data.amount * 100) * getRate(base, rates, data.from ?? from, data.to ?? to) / 100;
+		from = data.from ?? from;
+		to = data.to ?? to;
	}

-	return (amount * 100) * getRate(base, rates, from, to) / 100;
+	return new Big(amount).multiply(getRate(base, rates, from, to)).toNumber();
}
```

**Comment by xxczaki** [comment]
Sorry for the inactivity! I just released 3.0.0 with many improvements, including [integration with big.js](https://github.com/xxczaki/cashify#integration-bigjs) :smile:
